sdk/storage/azure-storage-blob/perf.yml

```diff
@@ -0,0 +1,32 @@
+parameters:
+  - name: packageVersions
+    displayName: PackageVersions (regex of package versions to run)
+    type: string
+    default: '12|source'
+  - name: tests
+    displayName: Tests (regex of tests to run)
+    type: string
+    default: '^(download|upload|list-blobs)$'
+  - name: arguments
+    displayName: Arguments (regex of arguments to run)
+    type: string
+    default: '(10240)|(10485760)|(1073741824)|(5 )|(500 )|(50000 )' 
```

Comment:
How does this match up download and upload vs listing? Is that was the space after `5`, `500`, `50000` is for?

Reply:
This is a regex selecting which test arguments to use.  First, the tests are filtered per the "tests" regex, then the arguments to each test are filtered per the "arguments" regex.  We can get away with a single regex for all tests, since the parameters are distinct sets.  Again this might need a better design going foward, where each test gets its own arguments regex.

Test arguments are also specified in the tools repo:

https://github.com/Azure/azure-sdk-tools/blob/main/tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/tests.yml#L101